en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md

```diff
@@ -65,21 +65,32 @@ Follow the  instructions below to set up a MySQL database:
 
 1.  Provide authorization to the user that you use to access the databases. 
 
-     For example, let's consider `apimadmin` as the user.
+     For example, let's consider `apimadmin` and `regadmin` as the users.
```

Comment:
shall we change the regadmin to something else? shared_db contains registry and user db. so only mentioning reg is not suitable i think

Reply:
@chamilaadhi how about sharedadmin? regadmin is used couple of places in this page. If we are changing will need to create another issue to do the same for all the other DB types as well.

Reply:
Created an issue for updating other Db types : https://github.com/wso2/docs-apim/issues/2570